lua/treesitter-context/render.lua

```diff
@@ -232,6 +232,16 @@ local function highlight_bottom(bufnr, row)
   })
 end
 
+---@param bufnr integer
+---@param row integer
+local function highlight_lno_bottom(bufnr, row)
```

Comment:
This is a bit repetitive. Can you combine it with the function above?

Reply:
I can add a third variable to the function above (e.g. a boolean telling me which hl_group to choose), if that is better?

Reply:
I would say add a string